internal/mtail/httpstatus.go

```diff
@@ -11,15 +11,21 @@ import (
 )
 
 const statusTemplate = `
+<!DOCTYPE html>
 <html>
 <head>
 <title>mtail on {{.BindAddress}}</title>
 </head>
 <body>
 <h1>mtail on {{.BindAddress}}</h1>
 <p>Build: {{.BuildInfo}}</p>
-<p>Metrics: <a href="/json">json</a>, <a href="/graphite">graphite</a>, <a href="/metrics">prometheus</a>, <a href="/varz">varz</a></p>
-<p>Debug: <a href="/debug/pprof">debug/pprof</a>, <a href="/debug/vars">debug/vars</a>, <a href="/tracez">tracez</a>, <a href="/progz">progz</a></p>
+<p>Metrics: <a href="/json">json</a>, <a href="/graphite">graphite</a>, <a href="/metrics">prometheus</a></p>
+<p>Info: {{ if .HTTPInfoEndpoints }}<a href="/varz">varz</a>, <a href="/progz">progz</a> <a href="/tracez">tracez</a></p>{{ else }} disabled {{ end }}</p>
+<p>Debug: {{ if .HTTPDebugEndpoints }}<a href="/debug/pprof">debug/pprof</a>, <a href="/debug/vars">debug/vars</a>{{ else }} disabled {{ end }}</p>
+`
+const statusTemplateEnd = `
```

Comment:
Why does this need to be a separate template?

Reply:
It just closes the HTML to be "properly" formatted. As we have conditional rendering of the status and runtime stats tables (depending on command line flags), adding this closing to the status and runtime rendering looks more complex to me. So this looks like the easiest path for me.

Reply:
Fixed the linter error.

Reply:
Ah yep I see why it needs to be like this now.  Thanks!